Apps/Database/src/Migrations/20191024233502_InitialCreate.cs

```diff
@@ -457,3 +460,7 @@ protected override void Down(MigrationBuilder migrationBuilder)
         }
     }
 }
+
+
+#pragma warning restore SA1200 // Using directive should appear within a namespace declaration
```

Comment:
will we ever need to regen these initial create files and so if so will we lose these code-smell-avoiding pragmas?

Reply:
Correct. Fortunately it looks like we only need to add // <auto-generated/> to the first line of the files